tools/cardinal/data_exporter.xml

```diff
@@ -1,4 +1,4 @@
-<tool id="cardinal_data_exporter" name="MSI data exporter" version="@VERSION@.0">
+<tool id="cardinal_data_exporter" name="MSI data exporter" version="@TOOL_VERSION@+galaxy@VERSION_SUFFIX@">
```

Comment:
please add also a profile="22.05" to each tool

Reply:
thanks, added the profile to all tools 👍